modules/database/variables.tf

```
@@ -26,3 +26,23 @@ variable "deletion_protection" {
   type        = bool
   default     = true
 }
+
+variable "apply_pitr_backup_rds" {
+  description = "Override default behaviour and restore from a point in time backup. Must be used with restore_time"
+  type        = bool
+  default     = false
+
+}
+
+variable "pitr_restore_time" {
```

We can make this a bit more verbose and make it clearer which nested parameter we are using (i.e. `restore_to_point_in_time` + `restore_time`) so that it's easier for users to reference this in the Terraform docs

```suggestion
variable "rds_restore_to_point_in_time_restore_time" {
```